### ISSUE TITLE
[Feature] Mouse hover over scrollbar should scroll that element.

### ISSUE THREAD

**Issue by FlamingFox911** [MurhafSousli/ngx-scrollbar#168]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
- [ ] question
```

### OS and Version?

Windows 10.

### Versions

Angular CLI: 8.0.4
Node: 12.4.0
OS: win32 x64
Angular: 8.0.2

### Repro steps

Hover over the scrollbar with the mouse and attempt to use scroll wheel.

### Desired functionality

The expected outcome needs to be that if you hover over a scrollbar, it should scroll that element.

**Comment by MurhafSousli** [comment]
The problem is the custom scrollbar block scroll view the scroll event loses focus, I tried to force focusing on the view when on scrollbar `mouseover` event but it didn't help.

Do you have any idea or workaround for it?

**Comment by FlamingFox911** [comment]
Not in the immediate sense.

I am rather new to Angular, but perhaps its possible to apply the `scrollLeft` & `scrollTop` properties of a child when you perform the scroll when mousing over the scroll bar using DOM?

There is something called a `WheelEvent` ([link](https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent)) in the DOM event objects that allow you to capture the delta x and y of a mouse wheel. Combining that with the `mouseover` might have the desired effect.

**Comment by MurhafSousli** [comment]
I think I figure it out, add `pointer-events: none;` on `.ng-scrollbar-visible` class would fix in compact mode

**Comment by FlamingFox911** [comment]
> I think I figure it out, add pointer-events: none; on .ng-scrollbar-visible class would fix in compact mode
- @MurhafSousli 

Just tested this using Chrome's Dev Tools and this does have the desired effect. It works.

**Comment by MurhafSousli** [comment]
But it blocks dragging the thumb

**Comment by MurhafSousli** [comment]
This is fixed in v5.0.0
